### PR TITLE
TM-3907 align eas boxes

### PR DESCRIPTION
**Pull request by andrews-john**
stuff is mostly done, just need to align search button

### CODE REVIEW

**Comment by andrews-john** [comment]
A lot of testing stuff still in this btw!

File: src/Components/Agenda/EmployeeAgendaSearch/EmployeeAgendaSearch.jsx [hunk 1]
```diff
@@ -305,42 +305,54 @@ const EmployeeAgendaSearch = ({ isCDO, viewType }) => {
                 <div className="search-header">
                   Search For An Employee
                 </div>
-                <div className="eas-search-form-container">
-                  <label htmlFor="last-name-search" className="search-label">
+                <div className="testclass usa-width-one-whole empl-search-filters">
```
Comment (pveland621) [inline src/Components/Agenda/EmployeeAgendaSearch/EmployeeAgendaSearch.jsx:308]: `testclass` can be removed

File: src/sass/_employeeSearch.scss [hunk 2]
```diff
@@ -104,11 +191,12 @@
...
-      .react-daterange-picker {
-        margin-left: 25px;
-      }
-    }
+    // @media screen and (max-width: $screen-md-max) {
+    //   .react-daterange-picker {
+    //     margin-left: 25px;
+    //   }
+    // this is why the TED would shift to the left. leaving in for now to draw attention to it 
+    // why was it here originally? if we deem this not needed i'll delete}
```
Comment (pveland621) [inline src/sass/_employeeSearch.scss:199]: 🧼

Comment (SidelineCory24) [reply src/sass/_employeeSearch.scss:199]: @pveland621 did you verify John's comments on whether or not this styling was being used or not?

**Comment by SidelineCory24** [comment]
Let's keep the search button the same size, it's a little smaller now
dev branch size
image

size on your pr
image

**Comment by SidelineCory24** [comment]
@andrews-john need to address merge conflicts

File: src/sass/_employeeSearch.scss (hunk 2 above)
Comment (pveland621) [reply src/sass/_employeeSearch.scss:199]: did not, @andrews-john sorry - does this need to be kept in?

**Comment by elizabeth-jimenez** [comment]
> Let's keep the search button the same size, it's a little smaller now and the original alignment is off **dev branch size** image
> 
> **size on your pr** image

This is intentional, check out how the filters collapse

File: src/sass/_employeeSearch.scss [hunk 3]
```diff
@@ -40,20 +40,103 @@
...
+        .search-results-inputs{
+          @media screen and (max-width: 1400px) {
+            padding: 10px 0px;
+          }
+        }
+
+        button {
+          width: 110px;
+          min-width: 100px;
+          padding: 8px 16px;
```
Comment (elizabeth-jimenez) [inline src/sass/_employeeSearch.scss:119]: padding: 12px 16px keeps the button at the same height

File: src/sass/_employeeSearch.scss (hunk 2 above)
Comment (SidelineCory24) [reply src/sass/_employeeSearch.scss]: I don't see any reason to keep this styling, neither of these look good lol. Clear to remove @andrews-john 
image

image

Comment (SidelineCory24) [reply src/sass/_employeeSearch.scss]: nice catch!

File: src/sass/_employeeSearch.scss [hunk 4]
```diff
@@ -40,20 +40,103 @@
...
+        display: none;
+      }
+    }
+    @media screen and (max-width: 1400px) {
+      display: grid;
+      grid-template-columns: repeat(auto-fill, minmax(380px, 1fr));
+    }
+
+    .search-submit-button{
+      margin: 0px 0px 0px 0px;
```
Comment (elizabeth-jimenez) [inline src/sass/_employeeSearch.scss:72]: Shorthand can be `margin: 0px`

**Comment by elizabeth-jimenez** [comment]
image
we'll want to make sure these are still visible at resolutions as low as 1100px

**Comment by elizabeth-jimenez** [comment]
> image we'll want to make sure these are still visible at resolutions as low as 1100px and the labels shouldn't be stacking

Ugh, ignore me! I think it was still looking at dev code

File: src/sass/_employeeSearch.scss [hunk 5]
```diff
@@ -40,20 +40,103 @@
...
+      display: inline-block;
+    }
   }
 
   .empl-search-filters {
     display: grid;
     grid-template-columns: repeat(auto-fill, minmax(350px, 1fr));
+    
+    @media screen and (max-width: 1685px) {
+      display: grid;
```
Comment (elizabeth-jimenez) [inline src/sass/_employeeSearch.scss:54]: you don't need to repeat `display:grid`, you would only have to define it in the media query if you wanted to overwrite the non-media query styling

File: src/sass/_employeeSearch.scss [hunk 6]
```diff
@@ -40,20 +40,103 @@
...
+
     .filter-div {
       align-items: center;
       display: flex;
       margin-right: 10px;
+
       .label {
         margin-right: 10px;
-        min-width: 90px;
+        min-width: 95px;
```
Comment (elizabeth-jimenez) [inline src/sass/_employeeSearch.scss:82]: 🙌

**Comment by elizabeth-jimenez** [comment]
check out how it jumps from 1685px to 1700 px - we'll want to add another media query or adjust the existing ones

File: src/sass/_employeeSearch.scss [hunk 7]
```diff
@@ -40,20 +40,103 @@
...
     display: grid;
     grid-template-columns: repeat(auto-fill, minmax(350px, 1fr));
+    
+    @media screen and (max-width: 1685px) {
+      display: grid;
+      grid-template-columns: repeat(auto-fill, minmax(325px, 1fr));
+    }
+    @media screen and (max-width: 1585px) {
+      display: grid;
+      grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
```
[the remark this answers is not in the document]
Comment (elizabeth-jimenez) [reply src/sass/_employeeSearch.scss:59]: Delete everything (except for your media-query for `fa-search`) from L52-70

[the remark this answers is not in the document]
Comment (elizabeth-jimenez) [reply src/sass/_employeeSearch.scss:59]: image
I suspect 🤔 you may have adjusted the media queries to handle this case? If that's the case (🥁😭), great job in trying to handle it! To handle it with the deletions mentioned above, adjust the original grid by increasing L51 from 350 to 370?ish

File: src/sass/_employeeSearch.scss [hunk 8]
```diff
@@ -40,20 +40,103 @@
...
+        font-size: 16px;
         text-align: end;
       }
+
+      .emp-search-div {   
+        input {
+          width: 230px;
+          @media screen and (max-width: 1475px) {
+            width: 170px;
+            max-width: 200%;
```
Comment (elizabeth-jimenez) [inline src/sass/_employeeSearch.scss:92]: what's the motivation behind this `max-width`?

File: src/sass/_employeeSearch.scss [hunk 9]
```diff
@@ -40,20 +40,103 @@
...
+          }
+        }
+        .search-results-inputs{
+          @media screen and (max-width: 1400px) {
```
Comment (elizabeth-jimenez) [inline src/sass/_employeeSearch.scss:96]: Why trigger this padding adjustment at 1400?

File: src/sass/_employeeSearch.scss [hunk 10]
```diff
@@ -40,20 +40,103 @@
...
+            padding: 10px 0px;
+          }
+        }
+
+        button {
+          width: 110px;
+          min-width: 100px;
```
Comment (elizabeth-jimenez) [inline src/sass/_employeeSearch.scss:118]: width overrides `min-width`. `min-width` is meant for an element that has the potential to re-size - defining a set width on it L117 overrides that

File: src/sass/_employeeSearch.scss [hunk 11]
```diff
@@ -40,20 +40,103 @@
...
+          padding: 8px 16px;
+          margin-left: 15px;
+          align-content: flex-end;
```
Comment (elizabeth-jimenez) [inline src/sass/_employeeSearch.scss:121]: image
From devtools

File: src/sass/_employeeSearch.scss [hunk 12]
```diff
@@ -40,20 +40,103 @@
...
+          }
+          @media screen and (max-width: 1585px) {
+            margin-left: 15px;
+            padding: 7px 10px;
+          }
+          @media screen and (max-width: 1475px) {
+            margin-left: 50px;
+          }
+          @media screen and (max-width: 1200px) {
+            margin-left: 70px;
```
Comment (elizabeth-jimenez) [inline src/sass/_employeeSearch.scss:135]: Let's consolidate this to 2 styles: ones for Button with icon and others for Button without icon

**Comment by elizabeth-jimenez** [comment]
Functionality almost good-to-go, mostly just a lot of extra code we can go ahead and remove 🎊

File: src/sass/_employeeSearch.scss (hunk 7 above)
[the remark this answers is not in the document]
Comment (andrews-john) [reply src/sass/_employeeSearch.scss:59]: Thats what i was trying to do lol, also why I shrank the search button

File: src/Components/Agenda/EmployeeAgendaSearch/EmployeeAgendaSearch.jsx [hunk 13]
```diff
@@ -305,42 +305,54 @@ const EmployeeAgendaSearch = ({ isCDO, viewType }) => {
...
+                        submitSearch={submitSearch}
+                        onChange={setSearchInputFirstName}
+                        ref={searchFirstNameRef}
+                        placeHolder="Search by First Name"
+                        textSearch={searchTextFirstName}
+                        noButton
+                      />
+                    </div>
+                  </div>
+                  <div className="filter-div split-filter-div">
```
Comment (elizabeth-jimenez) [inline src/Components/Agenda/EmployeeAgendaSearch/EmployeeAgendaSearch.jsx:341]: `split-filter-div` needed?

File: src/sass/_employeeSearch.scss (hunk 8 above)
Comment (andrews-john) [reply src/sass/_employeeSearch.scss:92]: I think it was a bandaid fix for something that's been resolved. Commented it out and I havent noticed anything different, so I'll get rid of it.

File: src/sass/_employeeSearch.scss (hunk 11 above)
Comment (andrews-john) [reply src/sass/_employeeSearch.scss:121]: Flex-End was a fix before I manually set the button's margins

File: src/sass/_employeeSearch.scss (hunk 9 above)
Comment (andrews-john) [reply src/sass/_employeeSearch.scss:96]: switched to 1395

**Comment by andrews-john** [comment]
Made a lot of these changes, but my local environment stopped working and I'm not positive if the css was updating or not. Going to undo a lot of these and go back through in the morning

File: src/sass/_employeeSearch.scss (hunk 8 above)
Comment (andrews-john) [reply src/sass/_employeeSearch.scss:92]: Max width is preventing the input boxes from being too short for the text inside (first name, last name) 
image

File: src/sass/_employeeSearch.scss (hunk 10 above)
Comment (andrews-john) [reply src/sass/_employeeSearch.scss:118]: min-width is keeping this search button at 78x16px. when commented out, it extends past the screen significantly
image

File: src/sass/_employeeSearch.scss (hunk 12 above)
Comment (andrews-john) [reply src/sass/_employeeSearch.scss:135]: Ive consolidated this as much as I can in the newest commit. the sections at 1200 and 1475 are needed to keep the padding consistent.

File: src/sass/_employeeSearch.scss (hunk 10 above)
Comment (elizabeth-jimenez) [reply src/sass/_employeeSearch.scss:106]: Discussed off github: inherited max and min widths throughout code causing issues. Clearing those `min-width: none;` should be the fix.

**Comment by SidelineCory24** [comment]
> @andrews-john need to address merge conflicts

@andrews-john flagging this for you again

File: src/sass/_employeeSearch.scss (hunk 8 above)
Comment (elizabeth-jimenez) [reply src/sass/_employeeSearch.scss]: Discussed off Github: issue was that a max-width was being inherited.

File: src/sass/_employeeSearch.scss (hunk 3 above)
Comment (elizabeth-jimenez) [reply src/sass/_employeeSearch.scss:107]: > padding: 12px 16px keeps the button at the same height

image
i don't think you ever addressed this one

**Comment by elizabeth-jimenez** [comment]
it's collapsing really beautifully!

**Comment by elizabeth-jimenez** [comment]
https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2442#discussion_r1096326710 for this one, it looked a little weird being that tall, so I added some margin instead to center it. @andrews-john I also made some other changes to simplify things. I'll go ahead and approve but i'll let the merge to you, dependent on if you agree with the changes

**Review by elizabeth-jimenez — APPROVED** [review]

**Comment by andrews-john** [comment]
Looks good, I like the changes. I'll merge them and put a deploy out

**Pull request merged**